README.md

```diff
@@ -4,6 +4,19 @@ related to infrastructure automation and CI/CD.
 
 Please feel free to raise bugs or PRs to improve functionality.
 
+# Credits
+
+Sponsored by nearForm.
+
+Contributors:
+- David Gonzalez
+- Damian Beresford
+- Matteo Collina
+- Luca Maraschi
+- Valerio Barrila
```

Comment:
nearformEmails? If not, I will merge it and raise PR with the correct emails (or send me the list on slack).